vendor/github.com/go-openapi/swag/string_bytes.go

```diff
@@ -0,0 +1,8 @@
+package swag
+
+import "unsafe"
+
+// hackStringBytes returns the (unsafe) underlying bytes slice of a string.
+func hackStringBytes(str string) []byte {
+	return unsafe.Slice(unsafe.StringData(str), len(str))
```

Comment:
do we know where and how this is used? just want to make sure nothing is sharing byte data in ways that break our use of shared openapi objects

Reply:
I see one instance used in IsEqualFoldIgnoreSpace https://github.com/go-openapi/swag/blob/master/split.go#L425 

```
// isEqualFoldIgnoreSpace is the same as strings.EqualFold, but
// it ignores leading and trailing blank spaces in the compared
// string.
```

The function looks to perform only read operations on the byte array.